content/docs/contributing/to-docs/templates/component-reference-doc.md

```diff
@@ -0,0 +1,428 @@
+
+* Crop the image to show only the component card of this component.
+
+* [Store the image files](#storing-image-files) in the correct folder. 
+
+* Follow the guidelines in [Submitting Media to Open 3D Engine Documentation](/docs/contributing/to-docs/style-guide/media/). 
+
+
+#### Creating one or multiple property tables
+
+Depending on how properties are arranged in your component card, you can have one or multiple property tables. 
+
+A *simple set of properties* refers to a component card that displays all of its properties at the first level in the component card's content hierarchy. In this case, you can view all of the properties without opening a drop down list or configuring a set of properties a specific way. 
+
+On the other hand, a *complex set of properties* doesn't show all available properties immediately. There may be properties hidden in a property group under a drop down list or conditional properties that only appear when other properties are configured a specific way. 
+
+| Example: Simple set of properties | Example: Complex set of properties | 
+| - | - |
+| ![Box Shape component properties](/images/user-guide/components/reference/shape/box-shape-component-ui-01.png) | ![PhysX Collider component interface.](/images/user-guide/components/reference/physx/physx-collider-ui-01.png) |
+
+{{< tabs >}} 
+{{% tab name="Simple set of properties"%}}
+
+For a component card with a simple set of properties, you only need one property table. 
+
+**Examples**:
+
+* [Box Shape component](http://localhost:1313/docs/user-guide/components/reference/shape/box-shape/): A straightforward example where all properties are displayed at the first level in the component card's content hierarchy.
+
+* [Bloom component](/docs/user-guide/components/reference/atom/bloom/): Even though the Bloom component's properties contains dropdown lists, only one property table is needed. This is because the drop down lists are opened by default and are used to contain an array of values. Another exception is the **Overrides** dropdown list, which isn't used in the user's main workflow. So, only a description of the **Overrides-Enabled Override** property is sufficient. 
+
+See `template-component-reference.md`:
+```md
+
+## Properties
+
+![<component name> interface](/images/<path-to-images>)
+
+| Property | Description | Value | default |
+| - | - | - | - |
+|   |   |   |   |
+|   |   |   |   |
+|   |   |   |   |
+
+```
+
+{{% /tab %}}
+{{% tab name="Complex set of properties"%}}  
+
+For a component card with a complex set of properties, you may need multiple property tables: base properties, and one for each property group or configuration. The first property table is called "Base properties" and contains all of the properties that are displayed at the first level in the component card's content hierarchy. You can name subsequent property tables after the property group name in the component card or the type of configuration. 
```

Comment:
You might want to mention when and how **tabs** might be needed in their property documentation, as shown in the Camera Rig component. Then you could link to the section on [tab shortcodes](https://o3de.org/docs/contributing/to-docs/style-guide/format/#tabs) in the style guide for details on how to create them in Markdown.

Reply:
Added!